### ISSUE TITLE
Add-PnPGroupMember does not work for "Everyone except external users"

### ISSUE THREAD

**Issue by DmitryVorobyov** [pnp/powershell#202]
### Reporting an Issue or Missing Feature
Please confirm what it is that your reporting

Bug: Add-PnPGroupMember does not work for "Everyone except external users". The code below used to work with Add-PnPUserToGroup cmdlet, it throws an error now.

### Expected behavior 

"Everyone except external users" is added to the specified SharePoint group (e.g. Visitors)

### Actual behavior 

Error message: 
_Add-PnpGroupMember: Exception has been thrown by the target of an invocation._

![Add-PnpUserToGroup_Everyone_Error](https://user-images.githubusercontent.com/10333583/106487140-5e8fb580-6480-11eb-87de-95183adb0209.png)

### Steps to reproduce behavior

```
Connect-PnPOnline $siteUrl -UseWebLogin

$ctx = Get-PnPContext
$web = Get-PnPWeb -Includes WebTemplate, Configuration

$Principal = [Microsoft.SharePoint.Client.Utilities.Utility]::ResolvePrincipal($ctx, $web, "Everyone except external users", "All", "All", $Null, $True)
$ctx.ExecuteQuery()
$everyoneLoginName = $Principal.Value.LoginName

Add-PnPGroupMember -LoginName $everyoneLoginName -Identity $web.AssociatedVisitorGroup
```

### What is the version of the Cmdlet module you are running?
1.2.0

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by erwinvanhunen** [comment]
Behind the scenes we are not using CSOM anymore in this cmdlet to acquire the correct group, for that reason we require the group to have the ID property loaded before we can add a member to it.

In a more 'powershell' way, this works with the release you have installed:

```powershell
$user = Get-PnPUser | ?${$_.Title -eq "Everyone except external users"} # but your approach to find the principal will work too
$group = Get-PnPGroup -AssociatedVisitorGroup
Add-PnPGroupMember -Login $user.LoginName -Group $group
```

We will merge an update today which will actually resolve the id of the group if not specified before we try to add the user to group. This change will be available in release 1.2.11-nightly.

**Comment by DmitryVorobyov** [comment]
Thank you!
